lupa/_lupa.pyx

```diff
@@ -1399,12 +1402,13 @@ cdef bint call_python(LuaRuntime runtime, lua_State *L, py_object* py_obj) excep
 
 cdef int py_call_with_gil(lua_State* L, py_object *py_obj) with gil:
     cdef LuaRuntime runtime = None
+    runtime = <LuaRuntime?>py_obj.runtime
     try:
-        runtime = <LuaRuntime?>py_obj.runtime
         return call_python(runtime, L, py_obj)
     except:
-        try: runtime.store_raised_exception()
```

Comment:
Well, what is the worst thing that can happen? If the outer calling code in Lupa detects a Lua error without a Python exception returned, then it will raise a LuaError. Given that this will really only happen in extreme cases (when running out of memory), this seems good enough to me.